doc/classes/FileDialog.xml

```diff
@@ -83,6 +83,10 @@
 			If [code]true[/code], the dialog will show hidden files.
 		</member>
 		<member name="title" type="String" setter="set_title" getter="get_title" overrides="Window" default="&quot;Save a File&quot;" />
+		<member name="use_native_dialog" type="bool" setter="set_use_native_dialog" getter="get_use_native_dialog" default="false">
+			If [code]true[/code], [member access] is set to [code]ACCESS_FILESYSTEM[/code], and it is supported by the current [DisplayServer], OS native dialog will be used instead of custom one.
```

Comment:
```suggestion
			If [code]true[/code], [member access] is set to [constant ACCESS_FILESYSTEM], and it is supported by the current [DisplayServer], OS native dialog will be used instead of custom one.
```